tests/simulation/test_circuit.py

```diff
@@ -95,3 +106,18 @@ def test_alternative_apply_cnot_three_qubits(BackendClass):
     prop_exp = np.array([0, 0, 0, 0, 0, 0, 0, 1**2])
     assert all(abs(abs(prop_found[i]) - abs(prop_exp[i])) < 0.2 for i in range(8)), \
         f"Propagation failed. Expected prop {prop_exp} vs. found {prop_found}."
+    
+def test_non_close_gate_binary_circuit():
+    n_qubit = 3
+    psi0 = [1] + [0] * (2**n_qubit-1)
+    qubit_layout = [0,1,14]
+    circ = BinaryCircuit(nqubit = n_qubit, depth=1, gates= standard_gates)
```

Comment:
@paolodr98 Do whitespaces before and after "=" in the case it is used with keyword arguments.